14.0.Dockerfile

```
@@ -0,0 +1,185 @@
+FROM python:3.7-slim-buster AS base
+
+EXPOSE 8069 8072
+
+# Enable Odoo user and filestore
+RUN useradd -md /home/odoo -s /bin/false odoo \
+    && mkdir -p /var/lib/odoo \
+    && chown -R odoo:odoo /var/lib/odoo \
+    && sync
+
+# System environment variables
+ENV GIT_AUTHOR_NAME=docker-odoo \
+    GIT_COMMITTER_NAME=docker-odoo \
+    EMAIL=docker-odoo@example.com \
+    LC_ALL=C.UTF-8 \
+    NODE_PATH=/usr/local/lib/node_modules:/usr/lib/node_modules \
+    PATH="/home/odoo/.local/bin:$PATH" \
+    PIP_NO_CACHE_DIR=0 \
+    PYTHONOPTIMIZE=1
+
+# Default values of env variables used by scripts
+ENV ODOO_SERVER=odoo \
+    UNACCENT=True \
+    PROXY_MODE=True \
+    WITHOUT_DEMO=True \
+    WAIT_PG=true \
+    PGUSER=odoo \
+    PGPASSWORD=odoo \
+    PGHOST=db \
+    PGPORT=5432 \
+    ADMIN_PASSWORD=admin
+
+# Other requirements and recommendations to run Odoo
+# See https://github.com/$ODOO_SOURCE/blob/$ODOO_VERSION/debian/control
+ARG WKHTMLTOPDF_VERSION=0.12.5
+ARG WKHTMLTOPDF_CHECKSUM='1140b0ab02aa6e17346af2f14ed0de807376de475ba90e1db3975f112fbd20bb'
+RUN apt-get -qq update \
+    && apt-get install -yqq --no-install-recommends \
+        curl \
+    && curl -SLo wkhtmltox.deb https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/${WKHTMLTOPDF_VERSION}/wkhtmltox_${WKHTMLTOPDF_VERSION}-1.stretch_amd64.deb \
+    && echo "${WKHTMLTOPDF_CHECKSUM}  wkhtmltox.deb" | sha256sum -c - \
+    && apt-get install -yqq --no-install-recommends \
+        ./wkhtmltox.deb \
+        chromium \
+        ffmpeg \
+        fonts-liberation2 \
+        gettext-base \
+        git \
+        gnupg2 \
+        locales-all \
+        nano \
+        npm \
+        wget \
+        openssh-client \
+        telnet \
+        vim \
+        zlibc \
+        sudo \
+    && echo 'deb http://apt.postgresql.org/pub/repos/apt/ stretch-pgdg main' >> /etc/apt/sources.list.d/postgresql.list \
+    && curl -SL https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add - \
+    && apt-get update \
+    && apt-get install -yqq --no-install-recommends postgresql-client \
+    && apt-get autopurge -yqq \
+    && rm -Rf wkhtmltox.deb /var/lib/apt/lists/* /tmp/* \
+    && sync
+
+ARG ODOO_VERSION=13.0
```

```suggestion
ARG ODOO_VERSION=14.0
```

si bien se sobreescribe con el hook.. por consistencia